docs/Manifests.md

```diff
@@ -1,6 +1,190 @@
 # Manifests
 
```

Comment:
An introduction section for the concepts of Manifests might be a good fit before getting into the `AppManifest Specification`

Reply:
Definitely - understood both your comments. So break the specs into two separate files and use Manifests as a way to give a more general explanation of what a manifest actually is. Cool, I think this would also bump up the headings by at least one and create a little bit more of a table of contents, at least for the primary attributes of components, resources, etc., in the spec.